tests/unit/scripting.tcl

```diff
@@ -616,6 +616,42 @@ start_server {tags {"scripting"}} {
              [run_script {return redis.sha1hex('Pizza & Mandolino')} 0]
     } {da39a3ee5e6b4b0d3255bfef95601890afd80709 74822d82031af7493c20eefa13bd07ec4fada82f}
 
+    test "Measures elapsed time os.clock()" {
+        set escaped [run_script {
+            local start = os.clock()
+            while os.clock() - start < 1 do end
+            return {double = os.clock() - start}
+        } 0]
+        assert_morethan_equal $escaped 1 ;# 1 second
+    }
+
+    test "Prohibit dangerous lua methods in sandbox" {
+        assert_equal "" [run_script {
+            local allowed_methods = {"clock"}
```

Comment:
let's add a simple test that tries to use a method that we blocked and verify it fails.
i.e. unlike listing the allowed methods, attacking this from the simpler usage attempt.

Reply:
Done with [`0ce727b` (#12971)](https://github.com/redis/redis/pull/12971/commits/0ce727baff8d3e67cca95c23f742d39989df2819), The first test was like this, but I replaced it with a new one.